README.md

```diff
@@ -62,6 +62,7 @@ git clone git@github.com:tforster/git-new.git
 
 ## Release History
 
+- 0.1.1 **Updated eslint dependency version** (2020-05-29)
```

Comment:
Should we use title case or sentence case for these lines?

Reply:
Should we really be bolding in the first place? Since we're creating a list, I don't think it needs to be a list of titles. Ergo, I favour sentence case, although I'm not sold on the overall formatting of the line and think we can improve that.

Reply:
Previously when creating features, the ticket names were short, like **Marketing Page**, so it made sense to use it as the title for the branch, where each commit was a sentence about what was changed.  Having the bold titles helped them stand out in a long list, where headings weren't appropriate (like the `CHANGELOG` that I had sent over previously).  I had also styled the version number, simply for a nicer format when scanning the list.  I am flexible though.  Here is an example with and without the formatting:

`v0.3.0` **Marketing Page** (2020-03-01)
- `#123` This is an example of the sentences below
- `#123` This is another example of the sentences below
- `#123` This is an example of the sentences below

`v0.2.0` **Home Page** (2020-02-01)
- `#123` This is an example of the sentences below
- `#123` This is another example of the sentences below
- `#123` This is an example of the sentences below

`v0.1.0` **Initial Release** (2020-01-01)
- `#123` This is an example of the sentences below
- `#123` This is another example of the sentences below
- `#123` This is an example of the sentences below

---

Which stands out more than:

---

v0.3.0 Marketing Page (2020-03-01)
- #123 This is an example of the sentences below
- #123 This is another example of the sentences below
- #123 This is an example of the sentences below

v0.2.0 Home Page (2020-02-01)
- #123 This is an example of the sentences below
- #123 This is another example of the sentences below
- #123 This is an example of the sentences below

v0.1.0 Initial Release (2020-01-01)
- #123 This is an example of the sentences below
- #123 This is another example of the sentences below
- #123 This is an example of the sentences below